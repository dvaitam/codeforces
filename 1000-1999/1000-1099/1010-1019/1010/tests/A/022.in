6
9
8 2 6 2 2 5
4 9 5 9 2 5
