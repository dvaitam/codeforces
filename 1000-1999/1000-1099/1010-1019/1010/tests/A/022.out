641.8928571429
